src/Kernel.php

```diff
@@ -5,9 +5,21 @@
 namespace App;
 
 use Symfony\Bundle\FrameworkBundle\Kernel\MicroKernelTrait;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
```

Comment:
Better to use imports in the generated code @ruudk , it makes the code easier to read IMO.

Reply:
You're right, but how to do that?

The code in question is commented out: 
https://github.com/twigstan/twigstan/blob/1c061819afb537b7d832eafbbd5c6c5bc3363687/example/twig-loader-symfony.php#L15-L26

So doing it like this felt easier , so that people can just copy and paste it. 

I could have also added a comment for the import, but that's extra friction. 

Ideally, this step won't be needed when we have a proper recipe for installation. Maybe a Symfony bundle.

Reply:
Right. Let's keep this as it is then.